web/client/plugins/FeaturedMaps.jsx

```diff
@@ -66,11 +66,15 @@ class FeaturedMaps extends React.Component {
                 title={<h3><Message msgId="manager.featuredMaps" /></h3>}
                 maps={items}
                 colProps={this.props.colProps}
-                viewerUrl={(map) =>
-                    map.category && map.category.name === "DASHBOARD"
-                        ? this.context.router.history.push(`/dashboard/${map.id}`)
-                        : this.context.router.history.push("/viewer/" + this.props.mapType + "/" + map.id)
-                }
+                viewerUrl={(map) => {
+                    if (map.category && map.category.name === "DASHBOARD") {
+                        return this.context.router.history.push(`/dashboard/${map.id}`);
+                    }
+                    if (map.category && map.category.name === "GEOSTORY") {
+                        return this.context.router.history.push(`/geostory/${map.id}`);
```

Comment:
this is not a map, is a story, maybe we can generalize the name "map" to resource.

Also the component name is should be named differently since it's handling  not only featured maps :)

Reply:
Yes It's not a map but also dashboards aren't maps, do you want to change the plugin name ora just the var name map?

Reply:
for now just the var name